docs/wearables/sdks/flutter.mdx

```diff
@@ -136,6 +136,27 @@ Finally, observe the status stream, so you can react to changes:
 Stream<SyncStatus> status = client.healthServices.status;
 ```
 
+### 3. Background Delivery
+
+Setting up background syncing in Flutter follows the [same approach as native iOS](/wearables/sdks/ios/). You need to configure `VitalHealthKit` in the AppDelegate, otherwise background syncing won't work.
+
+You can do so by calling:
+
+```swift
+import VitalHealthKit
+
+VitalHealthKitClient.configure(
```

Comment:
maybe wrap it in a `application(_:didFinishLaunchingWithOptions:)` dummy declaration

Reply:
Done!